tests/profiles-enabled/conftest.py

```diff
@@ -76,6 +81,46 @@ def shipper_user(shipper_token):
     return passive_credentials_auth(shipper_token)
 
 
+class JsonAPIClient(APIClient):
```

Comment:
Do we have this in python-common somewere? I feel like I've seen it somewhere else, maybe in the profiles codebase. So this probably should be moved to common

Reply:
We also use this in Profiles so we could move it to the test_utils